static/src/javascripts/es6/projects/common/modules/crosswords/main.js

```diff
@@ -375,6 +375,14 @@ class Crossword extends React.Component {
         }
     }
 
+    allHighlightedClues () {
+        return _.filter(this.props.data.entries, (entry) => {
```

Comment:
could be simplified to `_.filter(this.props.data.entries, this.clueIsInFocusGroup)`